content/en/docs/instrumentation/js/manual.md

```diff
@@ -376,16 +866,41 @@ const span = opentelemetry.trace.getSpan(ctx);
 pairs to a [`Span`](/docs/concepts/signals/traces/#spans) so it carries more
 information about the current operation that it's tracking.
 
-```javascript
-tracer.startActiveSpan('app.new-span', (span) => {
-  // do some work...
+<!-- prettier-ignore-start -->
+{{< tabpane langEqualsHeader=true >}}
 
-  // Add an attribute to the span
-  span.setAttribute('attribute1', 'value1');
+{{< tab TypeScript >}}
+function rollOnce(i: number, min: number, max: number) {
+  return tracer.startActiveSpan(`rollOnce:${i}`, (span: Span) => {
+    const result = Math.floor(Math.random() * (max - min) + min);
 
-  span.end();
-});
-```
+    // Add an attribute to the span
+    span.setAttribute('dicelib.rolled', result.toString());
+
+    span.end()
+    return result
+  })
+}
+
+{{< /tab >}}
+
+{{< tab JavaScript >}}
+function rollOnce(i, min, max) {
+  return tracer.startActiveSpan(`rollOnce:${i}`, (span) => {
+    const result = Math.floor(Math.random() * (max - min) + min);
+
+    // Add an attribute to the span
+    span.setAttribute('dicelib.rolled', result.toString());
+
+    span.end()
+    return result
+  })
+}
+
+{{< /tab >}}
+
+{{< /tabpane >}}
+<!-- prettier-ignore-end -->
```

Comment:
Recommend adding a short paragraph at the end stating that the key difference here is `startSpan`, which by default creates an independent span rather than a nested/child span.

Reply:
not sure what your comment is referring to? In the lines above no `startSpan` is used